workflow-templates/check-license.yml

```
@@ -59,13 +59,29 @@ jobs:
 
           echo "result=$RESULT" >> $GITHUB_OUTPUT
 
-  check-license:
+  build:
+    name: ${{ matrix.check-license.path }}
     needs: run-determination
     if: needs.run-determination.outputs.result == 'true'
     runs-on: ubuntu-latest
     permissions:
       contents: read
 
+    strategy:
+      fail-fast: false
+
+      matrix:
+        check-license:
+          #TODO Add additional paths where license files should be
```

```suggestion
          # TODO: Add additional paths where license files should be
```

Use standardized style for "TODO" comments. The person performing an installation of the template may be searching for the specific "`TODO:`" string to identify the places where project-specific modifications are necessary.

---

Could you share documentation on the standardized use of `TODO:`?

---

There is no documentation. It's simply an established convention used through the files in the repository.

When you are doing development work on a project you should always try to follow the established conventions for the sake of consistency. If a formal style guide isn't available, do a survey of the existing content of the project and extrapolate the convention from what you find there. That can be very frustrating in projects where sloppy development and maintenance has resulted in an utter mishmash of styles, but there is a pretty good level of consistency in this project so you should not have much trouble determining the common conventions here.